openmls/src/group/tests/kat_transcripts.rs

```diff
@@ -26,18 +26,43 @@ use serde::{self, Deserialize, Serialize};
 use thiserror::Error;
 use tls_codec::{Deserialize as TlsDeserialize, Serialize as TlsSerializeTrait};
 
+/// ```json
```

Comment:
Unrelated? This is probably not worth touching because we should change it. Anyway, just a comment.